modules/secure-ci/main.tf

```
@@ -78,7 +78,7 @@ resource "google_cloudbuild_trigger" "app_build_trigger" {
 module "gcloud" {
   source                            = "terraform-google-modules/gcloud/google"
   count                             = var.skip_provisioners ? 0 : 1
-  version                           = "~> 3.1.0"
+  version                           = "~> 3.3.0"
```

nit: I might suggest pessimistic pinning to a major rather than minor version to reduce the frequency of updates.

```suggestion
  version                           = "~> 3.1"
```